src/docfx/build/toc/TableOfContentsNode.cs

```diff
@@ -73,5 +73,12 @@ public TableOfContentsNode(TableOfContentsNode item)
             Children = item.Children;
             LandingPageType = item.LandingPageType;
         }
+
+        public TableOfContentsNode CloneWithoutItems()
```

Comment:
consider putting this into `TableOfContentsLoader`, do not feel like it is a common method here

Reply:
For `MemberwiseClone` is to create a shallow copy of the current Object.
It must be placed in the `TableOfContentsNode `class.